person.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Person < Nameable
```

Comment:
- [ ] It will be great to create a method `add_rental()` in this class in order to ensure the many-many relationship between the book and the person. Hint. ⬇️ 
```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```